pkgs/games/openttd/default.nix

```diff
@@ -32,11 +32,11 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "openttd";
-  version = "13.4";
+  version = "14.0";
 
   src = fetchurl {
     url = "https://cdn.openttd.org/openttd-releases/${version}/${pname}-${version}-source.tar.xz";
-    hash = "sha256-Kh3roBv+WOIYiHn0UMP6TzgZJxq0m/NI3WZUXwQNFG8=";
+    hash = "sha256-lvdquFiBal4wA4reBpLm6/NQufcL8Zx7SN2oRciEGLE=";
```

Comment:
Since `hash` is preferred, can you change `sha256` to `hash` of fetchzips in a seperate commit?

Reply:
So would switching from fetchurl to fetchzip be preferred? If so I'll add that to this PR.

Reply:
I meant changing the `sha256` attributes of  optional dependencies/build inputs like these to `hash`.
https://github.com/NixOS/nixpkgs/blob/913dd8c865457938202a9d92294a3ebe43fb3255/pkgs/games/openttd/default.nix#L18

Additionally, looks like OpenTTD has a [github repo](https://github.com/OpenTTD/OpenTTD), so for this change, instead of fetching the source from their website, it can be fetched from their GitHub repo using `fetchFromGitHub`.

Reply:
Something like this should work for main src.
```
  src = fetchFromGitHub {
    owner = "OpenTTD";
    repo = "OpenTTD";
    rev = "${version}";
    sha256 = "sha256-3EKGeIiYNwdV4Xnk0Rtqn1zsmAGvqbC4KKn9QfIii8U=";
  };
```

Reply:
Bravo. Nice work on confusing a first-time contributor.

Reply:
Obviously that was not my intention. I apologize, I'll be more gentle in future.